src/utils/api.js

```diff
@@ -15,4 +15,10 @@ const getURL = (config) => {
   return `${base}photos?${query}&client_id=${key}`;
 };
 
-export default getURL;
+export const getPhotoUrl = (photoId) => {
+  const base = process.env.REACT_APP_UNSPLASH_API_URL;
```

Comment:
you can declare this variable at the top of the file, it's being used by all your functions